solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -186,7 +186,7 @@
 
   public static final String version = "1.0";
 
-  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  protected static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
Reusing the logger object for the extended class would probably result in the log lines printing out o.a.s.SolrCore instead of o.a.s.SolrCoreProxy. Might be better to create another private log object in the SolrCoreProxy.